searchlib/src/vespa/searchlib/queryeval/blueprint.h

```diff
@@ -246,9 +260,9 @@ class Blueprint
     virtual void setDocIdLimit(uint32_t limit) noexcept { _docid_limit = limit; }
     uint32_t get_docid_limit() const noexcept { return _docid_limit; }
 
-    static Blueprint::UP optimize(Blueprint::UP bp);
-    virtual void optimize(Blueprint* &self, OptimizePass pass) = 0;
-    virtual void optimize_self(OptimizePass pass);
+    static Blueprint::UP optimize(Blueprint::UP bp, bool sort_by_cost);
+    virtual void optimize(Blueprint* &self, OptimizePass pass, bool sort_by_cost) = 0;
+    virtual void optimize_self(OptimizePass pass, bool sort_by_cost);
```

Comment:
Could perhaps make a struct OptimizeInfo containing OptimizePass and sort_by_cost.